.travis.yml

```diff
@@ -9,10 +9,6 @@ matrix:
     - go: tip
 
 install:
-  - rm -rf $GOPATH/src/gopkg.in/src-d
```

Comment:
why are you removing this? if you remove this lines the tests are being executed in a different path and testing the current master branch.